docs/conf.py

```diff
@@ -12,7 +12,7 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('..'))
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
Wouldn't `os.path.join` be better as  `os.path.dirname`?

Reply:
I don't really follow this.